front/src/routes/scene/style.css

```
@@ -78,3 +78,63 @@
+  visibility: hidden;
+  color: #cd201f;
+}
+
+/* Outlined square (like the previous fe-square) but drawn with a currentColor
+   border, so it always matches the "Arrêter" text color exactly. */
+.stopIcon {
+  display: inline-block;
+  width: 0.85em;
+  height: 0.85em;
+  margin-right: 0.4em;
+  border: 1.5px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)